charts/portal/templates/analytics-server/analytics-config.yaml

```diff
@@ -7,6 +7,12 @@ metadata:
     chart: {{ template "portal.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+  {{- if  .Values.global.additionalAnnotations }}
```

Comment:
It would be good to apply global annotation only if there is no  resource specific annotation. Something like below:
{{- if .Values.secret.annotations }}
      annotations:
        {{- range $key, $val := .Values.secret.additionalAnnotations }}
           {{ $key }}: "{{ $val }}"
        {{- end }}
      {{- else if .Values.global.additionalAnnotations }}
       annotations:
        {{- range $key, $val := .Values.global.additionalAnnotations }}
           {{ $key }}: "{{ $val }}"
        {{- end }}
    {{- end }}